package.json

```diff
@@ -105,6 +107,18 @@
     "sinon": "^6.3.4",
     "tempy": "^0.2.1",
     "url-search-params": "^1.1.0",
-    "webpack": "^4.19.1"
+    "webpack": "^4.19.1",
+    "workbox-background-sync": "file:./packages/workbox-background-sync",
```

Comment:
You mentioned out of band that this was required for Rollup to bundle everything, but I am not sure how nicely this is going to play with Lerna moving forward. (To be clear, I'm not sure one way or another.)

Can you get away with, e.g., using the [following option](https://github.com/rollup/rollup-plugin-node-resolve) in `rollup-plugin-node-resolve` instead?

```
// Any additional options that should be passed through
// to node-resolve
customResolveOptions: {
  moduleDirectory: 'js_modules'
}
```

Reply:
Seems to work fine, I'll switch to this.